### PR TITLE
Custom es cloud namespaces

### PR DESCRIPTION
**Pull request by S4NT14G0**

### CODE REVIEW

File: charts/elastic-cloud/templates/secrets.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: aws
-  namespace: {{ .Release.Namespace }}
+  name: elastic-cloud-aws-credentials
```
Comment (while-loop) [inline charts/elastic-cloud/templates/secrets.yaml]: This name should be generated via the helm installation name.

 ```suggestion
  name: {{ template "elastic-cloud.fullname" . }}-aws-credentials
```

By default, it'd evaluate to ` elastic-cloud-aws-credentials`.


And can be overrideen via `.Values.fullnameOverride` (see [_helpers.tpl](https://github.com/rhoai/helm-charts/blob/master/charts/elastic-cloud/templates/_helpers.tpl#L14))

File: charts/elastic-cloud/examples/multi-node.yaml [hunk 2]
```diff
@@ -8,8 +9,10 @@ backups:
 
 kibana:
   enabled: true
+  namespace:
   ingress:
     enabled: false
+    namespace:
```
Comment (while-loop) [inline charts/elastic-cloud/examples/multi-node.yaml]: should this ingress-level namespace inherit the namespace already given in `kibana.namespace`?

**Review by while-loop — CHANGES_REQUESTED** [review]

File: charts/elastic-cloud/templates/kibana_ingress.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ apiVersion: networking.k8s.io/v1beta1
 kind: Ingress
 metadata:
   name: elasticsearch-kb-http
```
Comment (while-loop) [inline charts/elastic-cloud/templates/kibana_ingress.yaml:5]: see https://github.com/rhoai/helm-charts/pull/2#discussion_r541097608 for distinct CRD metadata names.

File: charts/elastic-cloud/templates/kibana_deployment.yaml [hunk 4]
```diff
@@ -3,6 +3,7 @@ apiVersion: kibana.k8s.elastic.co/v1
 kind: Kibana
 metadata:
   name: elasticsearch
```
Comment (while-loop) [inline charts/elastic-cloud/templates/kibana_deployment.yaml:5]: see https://github.com/rhoai/helm-charts/pull/2#discussion_r541097608 for distinct CRD metadata names.

File: charts/elastic-cloud/templates/elasticsearch_ingress.yaml [hunk 5]
```diff
@@ -3,7 +3,7 @@ apiVersion: networking.k8s.io/v1beta1
 kind: Ingress
 metadata:
   name: elasticsearch-es-http
```
Comment (while-loop) [inline charts/elastic-cloud/templates/elasticsearch_ingress.yaml:5]: see https://github.com/rhoai/helm-charts/pull/2#discussion_r541097608 for distinct CRD metadata names.

File: charts/elastic-cloud/templates/elasticsearch_deployment.yaml [hunk 6]
```diff
@@ -2,5 +2,6 @@ apiVersion: elasticsearch.k8s.elastic.co/v1
 kind: Elasticsearch
 metadata:
   name: elasticsearch
```
Comment (while-loop) [inline charts/elastic-cloud/templates/elasticsearch_deployment.yaml:4]: see https://github.com/rhoai/helm-charts/pull/2#discussion_r541097608 for distinct CRD metadata names.

File: charts/elastic-cloud/templates/elasticsearch_backup.yaml [hunk 7]
```diff
@@ -3,6 +3,7 @@ apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: elasticsearch-snapshotter
```
Comment (while-loop) [inline charts/elastic-cloud/templates/elasticsearch_backup.yaml:5]: see https://github.com/rhoai/helm-charts/pull/2#discussion_r541097608 for distinct CRD metadata names.

File: charts/elastic-cloud/examples/multi-node.yaml (hunk 2 above)
Comment (S4NT14G0) [reply charts/elastic-cloud/examples/multi-node.yaml]: Ahh yeah.  I'll update this for es/kibana

**Review by while-loop — CHANGES_REQUESTED** [review]

**Review by while-loop — APPROVED** [review]

**Pull request merged**